lms/djangoapps/courseware/tabs.py

```diff
@@ -41,6 +41,17 @@ class CoursewareTab(EnrolledTab):
     is_default = False
     supports_preview_menu = True
 
+    def __init__(self, tab_dict):
```

Comment:
Out of curiosity, why do the `__init__` change vs just updating the link_func below?

Reply:
We needed to pass an argument to check for the flag (`if course_home_mfe_outline_tab_is_active`), and I believe it was technically incorrect to pass additional arguments to the property. I mirrored how this was handled in `DatesTab` within the same file.